src/arrays/ladderops.jl

```diff
@@ -37,7 +37,16 @@ function momentumop(n::Int)
     return scale(im/sqrt(2.), cop-cop')
 end
 
+squeeze_construct(a::AbstractQuMatrix,b::AbstractQuMatrix, z::Union(Complex64, Float64)) = 0.5*(z'*(a*b)-z*(a'*b'))
```

Comment:
Makes sense, but we can still call it `squeezingop` - multiple dispatch is our friend. Please change it to use `scale!` in order to avoid copies.